lib/test.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2012-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+const Server = require('karma').Server;
+
+// load config defaults
+require('./config');
+
+bedrock.events.on('bedrock.tests.run', function(state, callback) {
+  if(bedrock.test.shouldRunFramework('karma')) {
+    return run(state, callback);
+  }
+  callback();
+});
+
+function run(state, callback) {
```

Comment:
Use `async` instead of callback.

Reply:
Or at least return a promise if `async` isn't needed because you aren't awaiting anything.

Reply:
```js
function run(state) {
  return new Promise((resolve, reject) => {
    // do the things and call resolve() or reject()
  });
}